.github/workflows/lint_python.yml

```diff
@@ -0,0 +1,11 @@
+name: lint_python
+on: push
+jobs:
+  lint_python:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-python@v2
+      - run: pip install black flake8
+      - run: black --check . || true
```

Comment:
isn't this || true disabling the check?

Reply:
No.  The check is still being run:
```

Run black --check . || true
would reformat /home/runner/work/hyperopt/hyperopt/hyperopt/anneal.py
would reformat /home/runner/work/hyperopt/hyperopt/hyperopt/base.py
would reformat /home/runner/work/hyperopt/hyperopt/hyperopt/fmin.py
would reformat /home/runner/work/hyperopt/hyperopt/hyperopt/pyll/base.py
would reformat /home/runner/work/hyperopt/hyperopt/hyperopt/mongoexp.py
would reformat /home/runner/work/hyperopt/hyperopt/hyperopt/pyll_utils.py
would reformat /home/runner/work/hyperopt/hyperopt/hyperopt/rdists.py
would reformat /home/runner/work/hyperopt/hyperopt/hyperopt/tests/test_domains.py
would reformat /home/runner/work/hyperopt/hyperopt/hyperopt/tpe.py
Oh no! 💥 💔 💥
9 files would be reformatted, 46 files would be left unchanged.
```
It is just not failing the job if black fails.  This is like how [`allow_failures` works in Travis CI](https://docs.travis-ci.com/user/customizing-the-build/#jobs-that-are-allowed-to-fail).  "_This lets you add in experimental and preparatory builds, for example to test against runtime versions or configurations that you are not ready to officially support._"

Reply:
yes, sorry, this is what I meant. We do want it to fail, why would we run it otherwise?

Reply:
"to test against runtime versions or configurations that you are not ready to officially support"

Like running codespell in allow_failures mode until #755 lands and then removing to `|| true` to make the test mandatory after #755 lands.  `|| true` is a way to highlight the work that is still to be done.

Reply:
I see. I thought after https://github.com/hyperopt/hyperopt/pull/748, black was already applied to the whole codebase, so we wouldn't need the `|| true`. But apparently that's not the case? Can we pin a specific version of black (same for travis and GHA)? I don't think it's a good to have this bothering because a new version is released

Reply:
pip install black=20.8b1